views/prisonerProfile/prisonerPersonal/partials/professionalContacts.njk

```diff
@@ -5,6 +5,10 @@
       <p class="govuk-body govuk-!-margin-bottom-1">{{ contact.relationshipDescription }}</p>
     </div>
   {% endfor%}
+
+  {# <div class="govuk-!-margin-top-3">
+    <a href="professional-contacts" class="govuk-link govuk-!-font-size-19">View all professional contacts and their contact details</a>
+  </div> #}
```

Comment:
Did you mean to leave this commented out?

Reply:
Yeah, I thought until Dom has tested it maybe so the page could go out on it's own if it was ever blocking a release.